src/main/webapp/WEB-INF/view/ProfilePageServelt.java

```diff
@@ -0,0 +1,25 @@
+package codeu.controller;
```

Comment:
I think you want this in the `src/main/java/codeu/controller` directory, right?

Reply:
Just fixed it!